server.go

```diff
@@ -41,6 +41,13 @@ var log = logrus.New()
 
 var tracer = trace.GlobalTracer
 
+type VeneurServer interface {
```

Comment:
Ah, yes, my previous comment about getter naming especially applies if we're defining an interface type with those method names. These would be better without the `Get` prefix.